chapters/install/os/on-rasp-linux.md

```diff
@@ -1,21 +1,64 @@
 
-# Install Jina on Raspberry Pi and other Linux Systems
+# How To Install Jina Core on Raspberry Pi OS and other Linux Systems
 
+## Overview
 
-On Raspbian or other Linux systems with Python >= 3.7 installed, you can simply install Jina via:
+This article explains how to successfully set up Jina Core on a Raspberry Pi using Raspberry Pi OS or Linux system in just a few steps.
```

Comment:
or other Linux systems

Reply:
Can remove `successfully`? We wouldn't write an article to show you how to fail at setting it up after all...